signac_dashboard/views.py

```diff
@@ -62,41 +74,51 @@ def show_job(dashboard, jobid):
         g.jobs = dashboard._get_job_details([job])
         g.title = g.jobs[0]["title"]
         g.subtitle = g.jobs[0]["subtitle"]
+        session["context"] = "JobContext"
         return dashboard._render_job_view(default_view="grid")
 
 
-def get_file(dashboard, jobid, filename):
-    try:
-        job = dashboard.project.open_job(id=jobid)
-    except KeyError:
-        abort(404, "The job id requested could not be found.")
+def get_file(dashboard, filename, jobid=None):
+    if jobid is not None:
+        try:
+            job_or_project = dashboard.project.open_job(id=jobid)
+        except KeyError:
```

Comment:
This call can also raise a `LookupError`. We should catch both. https://docs.signac.io/projects/core/en/latest/api.html#signac.Project.open_job
```suggestion
        except (KeyError, LookupError):
```